ws/src/main/resources/demo-configs/db/sql/DemoConnection.hjson

```diff
@@ -1,11 +0,0 @@
-{
-  dbconfigs: [
-    {
-      name: DemoConnection
-      url: jdbc:h2:mem:DemoDB;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:create-demo-data.sql'
```

Comment:
Can we also remove `create-demo-data.sql`?

Reply:
removed the file from app. It will come from the demo-config jar